docs/onboarding/09-exposing-twins-over-graphql.md

```diff
@@ -0,0 +1,45 @@
+# Chapter 9: Exposing Digital Twins over GraphQL
+
+- [Why GraphQL?](https://www.apollographql.com/docs/intro/benefits/) - Explains the value proposition of GraphQL
+- [GraphQL Tutorial](https://www.apollographql.com/docs/tutorial/introduction/) - Walks through coding a GraphQL service. **STOP after [Chapter 4 (Write Mutation Resolvers)](https://www.apollographql.com/docs/tutorial/mutation-resolvers/)**. Apollo Explorer is not necessary to complete the tutorial. Browsing to localhost address provides means to execute requests.
+
+## Key Points
+
+- GraphQL is a declarative model for exposing data from multiple sources over a common protocol.
+- Data Sources facilitate operations against a specific source of data such as a database or api.
+- Resolvers leverage one or more data sources to fulfill a requested operation
+- Schema defines both the model/state (e.g. Room, Floor) exposed as well as the supported operations (GetRoomsByFloor)
+- Clients must leverage an SDK that translates GraphQL operation into a protocol specific implementation (i.e. HTTP) as opposed to directly invoking an HTTP endpoint.
+
+> Since Azure Digital Twins also follows a declarative model for defining twins and relationships and, twins are likely to co-exist with other non-twin application data; GraphQL is a natural fit for exposing the aggregate of this information over a common protocol.
+
+## Learning Exercises
+
+### Prerequisites
+
+Javascript/Node will need to be To complete the exercises using Apollo GraphQL framework. VS Code is suggested IDE for completing these challenges.

```

Comment:
Something missing here. Did you mean to say "...need to be _installed_"?

Reply:
good catch! fixed.